lib/ipmi_chassis.c

```diff
@@ -1712,6 +1712,205 @@ ipmi_chassis_set_bootflag_help()
 	get_bootparam_options("options=help", &set_flag, &clr_flag);
 }
 
+/*
+ * Sugar. Macros for internal use by bootdev_parse_options() to make
+ * the structure initialization look better
+ */
+#define BF1_OFFSET 0
+#define BF2_OFFSET 1
+#define BF3_OFFSET 2
+#define BF4_OFFSET 3
+#define BF_BYTE_COUNT 5
```

Comment:
Random thought: If they were like 
```
static const int BF1_OFFSET = 0
```

Then they would be contained within the scope of this function and also type safe.

But also good to leave as is.

Reply:
Good point

Reply:
Alas, it turned out that gcc5 (default for Ubuntu Xenial) can't use `static const int` for structure initialization. gcc8 can do that, but for compatibility I'll keep these as macros.